### ISSUE TITLE
Look and Feel / Format Issue for Emails requiring approval

### ISSUE THREAD

**Issue by bds002** [magfest/ubersystem#1364]
For:
https://classic.uber.magfest.org/uber/emails/pending

We have a new practice of approving Rams/Uber Emails before they go out. Which given past issues is a good thing.  However, looking at the list above, I get heartburn thinking that I might have missed and Email that we are should be sending to inform someone about something, registration, staffing, shifts, hotel rooms etc etc etc....

What I would like to see is 2 fold:
1 - A display that sorts these Emails by department.  - This shouldn't be difficult since they are listed by Sender already, just not sorted into separate sections.
2. - Within each department sort by date from event they should be going out.  - 
----Or sort by the number of Emails being sent (discussing this with Eli, this option would make the page load a LOT slower, so I'm not sure that we want to add load time to things.)

**Comment by kitsuta** [comment]
Do we still need this before Classic?

**Comment by bds002** [comment]
I'm not sending out any more of these Emails, it would be good to have for Super Fest since there are 3X as many emails to go out.

**Comment by EliAndrewC** [comment]
Yeah, this is on my "soon" list, so this will be a fairly high priority after Classic, since I unfortunately didn't find time to implement it before then,

My first priority after Classic is probably the MIVS application stuff since that needs to be ready by Oct 1 and available for testing at least a week before that.  After that I plan to go to town on the backlog.
